### ISSUE TITLE
Month translation in archives for a non-english blog

### ISSUE THREAD

**Issue by thoas** [montylounge/django-mingus#28]
hi,

I have fixed an issue in [forked repository of django-basic-apps](http://github.com/thoas/django-basic-apps) when you haven't an English-based locale, the month is not caught by urls.

For example for august month, the render is "aoû" in French due to an automatic translation made by django.

My fix changes the month format to numeric type.

You also need to update this portion of code in the template _base.html_ of _django-mingus_:

```
{% cache 7200 "base.archive" %}
{% get_post_archive as dates %}
<h4>{% trans "Archive" %}</h4>
<ul class="archive">
{% for date in dates %}
<li><a href="{% url blog_archive_month date|date:"Y", date|date:"b" %}" title="{{ date|date:"F Y" }}">{{ date|date:"F Y" }}</a></li>
{% endfor %}
</ul>
{% endcache %}
```

to

```
{% cache 7200 "base.archive" %}
{% get_post_archive as dates %}
<h4>{% trans "Archive" %}</h4>
<ul class="archive">
{% for date in dates %}
<li><a href="{% url blog_archive_month date|date:"Y", date|date:"m" %}" title="{{ date|date:"F Y" }}">{{ date|date:"F Y" }}</a></li>
{% endfor %}
</ul>
{% endcache %}
```

Thanks again for your great work.

**Comment by montylounge** [comment]
agreed. will fix this.

**Comment by montylounge** [comment]
resolved in dev branch. will be pushed with 0.9 ... view now handles legacy "b" formatting but imoving forward with "m" formating.

**Comment by thoas** [comment]
Thank you very much.

**Comment by montylounge** [comment]
resolved in 0.9alpha and moving forward.
